.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" 
+    directory: "ember-flight-icons/" 
+    versioning-strategy: increase
+    schedule:
+      interval: "daily"
+    allow: 
+      - dependency-name: "@hashicorp/*"
```

Comment:
Figured we may want to start adding design-system-components and design-system-tokens here